### PR TITLE
fix(ui/button,link): fix css color inherit

### PR DESCRIPTION
**Pull request by lxing4812**
## Checklist

---

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

---

### CODE REVIEW

File: packages/varlet-ui/src/button/button.less [hunk 1]
```diff
@@ -1,5 +1,6 @@
 :root {
   --button-default-color: #f5f5f5;
+  --button-default-color-text: #555;
```
Comment (haoziqaq) [inline packages/varlet-ui/src/button/button.less:3]: recommended change to `--button-default-text-color`, `link` same as this.

**Review by haoziqaq — COMMENTED** [review]
Don't forget  about dark mode themes

File: packages/varlet-ui/src/link/link.less [hunk 2]
```diff
@@ -16,7 +17,7 @@
   cursor: pointer;
   font-family: inherit;
   white-space: nowrap;
-  color: inherit;
+  color: var(--link-default-color);
```
Comment (haoziqaq) [inline packages/varlet-ui/src/link/link.less:20]: This is probably unnecessary.

Comment (lxing4812) [reply packages/varlet-ui/src/link/link.less]: done

**Comment by haoziqaq** [comment]
thx~

**Pull request merged**